inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -378,46 +376,42 @@ discretized Gamma.
 
 ## Results
 
-```{r, eval = params$use_epiparameter}
-si_epiparameter <- epiparameter::epidist(
-  params$epiparameter_pathogen,
-  "serial_interval"
+```{r, eval = params$use_epiparameter_database}
+si_epidist <- epiparameter::epidist_db(
+  disease =  params$disease_name,
+  epi_dist =  "serial_interval",
+  single_epidist = TRUE,
+  subset = is_parameterised
 )
-si_params <- si_epiparameter$param
-si_dist <- si_epiparameter$dist
-si_mean <- si_params[[1]]
-si_sd <- si_params[[2]]
+
+si_params <- get_parameters(si_epidist)
+si_dist <- family(si_epidist)
+si_mean <- get_parameters(si_epidist)[1]
+si_sd <- get_parameters(si_epidist)[2]
```

Comment:
There is no need to reuse `get_parameters()` here as you already extracted them and stored them in `si_params`. Instead I would write

```r
si_mean <- si_params["mean"]
si_sd <- si_params["sd"]
```

This will require calling fewer functions and is more readable than using numeric subsetting. 

If you want the resulting `si_mean` and `si_sd` to be unnamed you can use double brackets (`[[`) instead:

```r
si_mean <- si_params[["mean"]]
si_sd <- si_params[["sd"]]
```

Reply:
Addressed in https://github.com/epiverse-trace/episoap/pull/100/commits/4c5f1975ce9e28ed676c72c8e1403743fca8d7e4